### PR TITLE
API-1783: Conditionally skip tests to help with k8s 1.30

### PR DESCRIPTION
**Pull request by soltysh**
/assign @jsafrane @dgoodwin

### CODE REVIEW

**Comment by dgoodwin** [comment]
/hold

Actually looks like this may not hit them all.

File: pkg/test/ginkgo/cmd_runsuite.go [hunk 1]
```diff
@@ -615,6 +615,9 @@ func (o *GinkgoRunSuiteOptions) filterOutRebaseTests(restConfig *rest.Config, te
 		// compare https://github.com/kubernetes/kubernetes/pull/123405
 		// which changed the healthz handler name
 		`[sig-api-machinery] health handlers should contain necessary checks`,
+		`PersistentVolumes GCEPD.*`,
```
Comment (dgoodwin) [inline pkg/test/ginkgo/cmd_runsuite.go:618]: Down below this looks like a strings.Contains, I don't think that will work with the .* wildcard, just need to remove that.

File: pkg/test/ginkgo/cmd_runsuite.go [hunk 2]
```diff
@@ -615,6 +615,9 @@ func (o *GinkgoRunSuiteOptions) filterOutRebaseTests(restConfig *rest.Config, te
...
+		`Multi-AZ Cluster Volumes should schedule pods in the same zones as statically provisioned PVs`,
+		`[Driver: azure-disk] [Testpattern: Pre-provisioned PV.*`,
```
Comment (dgoodwin) [inline pkg/test/ginkgo/cmd_runsuite.go:620]: Same here.

**Comment by soltysh** [comment]
/override ci/prow/lint 
this will be fixed in https://github.com/openshift/origin/pull/28834 with a timeout bump

**Pull request merged**